extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -55,6 +55,8 @@ function ScrollerWithRef({children, loop, restingIndex, setRestingIndex}, ref) {
       container./* OK */ scrollLeft += container./* OK */ offsetWidth * by;
     },
   }));
+  // TODO: uncomment when build step is ready..
```

Comment:
Here's an idea. Let's actually create the `useStyles()` fake in the `.jss.js` file. It'd simply look like this:

```
export function useStyles() {
  return classes;
}
```

Then we can do a couple of things: 
1. All component modules would look exactly like we want them to look. When build step is ready, none of them would need to change.
2. We can shield `classes` export and thus reduce public surface of the `jss` file.

Reply:
I wish I had thought of this. on it!